base/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # CEPH BASE IMAGE
-# CEPH VERSION: Infernalis
-# CEPH VERSION DETAIL: 9.2.x
+# CEPH VERSION: Jewel
+# CEPH VERSION DETAIL: 10.2.x
 
-FROM ubuntu:14.04
+FROM ubuntu:trusty
 MAINTAINER Sébastien Han "seb@redhat.com"
 
-ENV ETCDCTL_VERSION v2.2.4
+ARG DEBIAN_FRONTEND=noninteractive
+ENV ETCDCTL_VERSION v2.0.10
```

Comment:
sorry, I believe I was basing this on an outdated version. I force-pushed a commit without this change.